style.css

```diff
@@ -0,0 +1,169 @@
+* {
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+.section-menu {
+  width: 100%;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  z-index: 99999;
+  background-color: #dededebb;
+  -webkit-backdrop-filter: blur(5px);
+  backdrop-filter: blur(5px);
+}
+
+.container {
+  border-bottom-left-radius: 80px;
+  background-color: #fff;
+}
+
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+header ul li,
+.socials > li {
+  list-style: none;
+  height: 20px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  text-align: center;
+}
+
+header ul,
+.socials {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-wrap: wrap;
+  flex-wrap: wrap;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  margin: 0;
+  padding: 0;
+  padding-top: 12px;
+}
+
+a {
+  text-decoration: none;
+}
+
+.myLogo {
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
+  padding-left: 24px;
+  line-height: 20px;
+  text-align: center;
+}
+
+header {
+  background-color: #fff;
+}
+
+.toggler-icon {
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  padding-right: 25px;
+}
+
+.font-style {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+}
+
+.main {
+  background-image: url(/images/header-shapes\ mobile@2x.svg);

```

Comment:
- [Optional] Consider adding a `.` at the beginning of your image URL so it doesn't break on some machines. It doesn't display properly on my machine until I added the `.`. Your URL should be `./images/header-shapes\ mobile@2x.svg`